apps/web/pages/v2/event-types/[type]/index.tsx

```diff
@@ -85,7 +86,19 @@ export type FormValues = {
 
 const querySchema = z.object({
   tabName: z
-    .enum(["setup", "availability", "apps", "limits", "recurring", "team", "advanced", "workflows"])
+    .enum([
+      "setup",
+      "availability",
+      "apps",
+      "limits",
+      "recurring",
+      "team",
+      "advanced",
+      "workflows",
+      "embed-preview",
```

Comment:
In the approach that I have pushed changes for, it is not required to pollute event-types with Embed info